src/responses/search-response.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import { SearchResponseHeader } from './search-response-header';
 import { SearchResponseDetails } from './search-response-details';
+import { SearchRequest } from './search-request';
 
 /**
  * The top-level response model when retrieving a response from the advanced search endpoint.
```

Comment:
QUESTION: Should this still say "from the advanced search endpoint"?

Reply:
Good catch -- will update this!